app/javascript/flavours/glitch/styles/monsterfork/components/status.scss

```diff
@@ -0,0 +1,8 @@
+.status__edit-notice {
+  & > span {
+    color: $dark-text-color;
+    line-height: normal;
+    font-style: italic;
+    font-size: 12px;
+  }
+}
```

Comment:
We should probably move this to another file... this isn't monsterfork, this is glitchcafe.
Though there is the advantage of being able to cherry-pick changes over easily...